app/models/item.rb

```diff
@@ -3,7 +3,15 @@ class Item < ActiveRecord::Base
 
   validates_presence_of :name, :price, :quantity, :shelf_life_days
 
-  def expires
-    Date.today + shelf_life_days.days
+  def expires_on
+    created_at + shelf_life_days.days
+  end
+
+  def self.valid_items
+    Item.all.select { |item| item.expires_on >= Date.today }
+  end
+
+  def price_in_dollars
+    Money.us_dollar(price).to_s
```

Comment:
This is perfectly fine how it is, but just so you know for future projects, there's a cool gem called [money-rails](https://github.com/RubyMoney/money-rails) that will take care of most of the money work for you.